jlm/hls/backend/rhls2firrtl/RhlsToFirrtlConverter.cpp

```diff
@@ -1937,85 +1960,83 @@ MLIRGenImpl::MlirGen(const llvm::lambda::node * lambdaNode)
 {
   struct circt::firrtl::PortInfo port = {
-    builder.getStringAttr("clk"),
-    circt::firrtl::ClockType::get(builder.getContext()),
-    circt::firrtl::Direction::In,
-    {},
-    location,
+    Builder_->getStringAttr("clk"), circt::firrtl::ClockType::get(Builder_->getContext()),
+    circt::firrtl::Direction::In,   {},
+    Builder_->getUnknownLoc(),
   };
   ports->push_back(port);
 }
 
 // Returns a PortInfo of unsigned IntType with width of 1
 void
-MLIRGenImpl::AddResetPort(::llvm::SmallVector<circt::firrtl::PortInfo> * ports)
+RhlsToFirrtlConverter::AddResetPort(::llvm::SmallVector<circt::firrtl::PortInfo> * ports)
 {
   struct circt::firrtl::PortInfo port = {
-    builder.getStringAttr("reset"),
-    circt::firrtl::IntType::get(builder.getContext(), false, 1),
-    circt::firrtl::Direction::In,
-    {},
-    location,
+    Builder_->getStringAttr("reset"), circt::firrtl::IntType::get(Builder_->getContext(), false, 1),
+    circt::firrtl::Direction::In,     {},
+    Builder_->getUnknownLoc(),
   };
   ports->push_back(port);
 }
 
 void
-MLIRGenImpl::AddMemReqPort(::llvm::SmallVector<circt::firrtl::PortInfo> * ports)
+RhlsToFirrtlConverter::AddMemReqPort(::llvm::SmallVector<circt::firrtl::PortInfo> * ports)
 {
   using BundleElement = circt::firrtl::BundleType::BundleElement;
 
   ::llvm::SmallVector<BundleElement> memReqElements;
   memReqElements.push_back(GetReadyElement());
   memReqElements.push_back(GetValidElement());
   memReqElements.push_back(BundleElement(
-      builder.getStringAttr("addr"),
+      Builder_->getStringAttr("addr"),
       false,
-      circt::firrtl::IntType::get(builder.getContext(), false, 64)));
+      circt::firrtl::IntType::get(Builder_->getContext(), false, 64)));
   memReqElements.push_back(BundleElement(
-      builder.getStringAttr("data"),
+      Builder_->getStringAttr("data"),
       false,
-      circt::firrtl::IntType::get(builder.getContext(), false, 64)));
+      circt::firrtl::IntType::get(Builder_->getContext(), false, 64)));
   memReqElements.push_back(BundleElement(
-      builder.getStringAttr("write"),
+      Builder_->getStringAttr("write"),
       false,
-      circt::firrtl::IntType::get(builder.getContext(), false, 1)));
+      circt::firrtl::IntType::get(Builder_->getContext(), false, 1)));
   memReqElements.push_back(BundleElement(
-      builder.getStringAttr("width"),
+      Builder_->getStringAttr("width"),
       false,
-      circt::firrtl::IntType::get(builder.getContext(), false, 3)));
+      circt::firrtl::IntType::get(Builder_->getContext(), false, 3)));
 
-  auto memType = circt::firrtl::BundleType::get(builder.getContext(), memReqElements);
+  auto memType = circt::firrtl::BundleType::get(Builder_->getContext(), memReqElements);
   struct circt::firrtl::PortInfo memBundle = {
-    builder.getStringAttr("mem_req"), memType, circt::firrtl::Direction::Out, {}, location,
+    Builder_->getStringAttr("mem_req"), memType, circt::firrtl::Direction::Out, {},
```

Comment:
You are mixing here. Either each argument on a new line, or all of them on a single line. First option is preferred.